src/System.IO.Pipes/src/System.IO.Pipes.csproj

```diff
@@ -7,21 +7,14 @@
   <PropertyGroup>
     <ProjectGuid>{16EE5522-F387-4C9E-9EF2-B5134B043F37}</ProjectGuid>
     <AssemblyName>System.IO.Pipes</AssemblyName>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
I think we're trying to keep such version numbers entirely out of the csproj files, and just setting it in the dir.props.